src/WaveManager.cpp

```diff
@@ -82,6 +83,11 @@ int WaveManager::getEnemies()
     return(enemyCount);
 }
 
+int WaveManager::getEnemiesAlive()
+{
+    return(aliveEnemyCount);
+}
+
```

Comment:
A getter for `aliveEnemyCount` was added so we don't have to loop over enemies every time we want a count of them.